opflexagent/gbp_ovs_agent.py

```diff
@@ -110,8 +121,12 @@ def __init__(self, root_helper=None, **kwargs):
         self.quitting_rpc_timeout = kwargs['quitting_rpc_timeout']
         # Initialize the Endpoint Manager.
         # TODO(ivar): make this component pluggable.
-        self.ep_manager = ep_manager.EndpointFileManager().initialize(
-            self.host, self.bridge_manager, kwargs)
+        if (self.hypervisor_type and
```

Comment:
when self.hypervisor_type is not None, it's also not equal to ofcst.HYPERVISOR_VCENTER so you can remove the first part of the check